### DISCUSSION TITLE
Show correct local time offset.

### DISCUSSION THREAD

**Discussion by real-yfprojects** [community/community#48585] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

There is this fairly new feature for setting the local timezone and displaying the local time on one's profile. Next to the local time, the local timezone is displayed slightly greyed out.

My first critique would be that my local timezone is shown as `UTC +01:00` while other timezones are displayed differently as `5h30m ahead`. However this design decision also has a big advantage. It allows me to see the time difference between me and the other user.

There is one problem though. The offset is calculated based on UTC instead of my local time. When reading `5h30m ahead` i would think the time would be 5h30m *ahead of me*. But actually the other user is only 4h30m ahead of me. Showing the latter would definitely be not only more intuitive but also more useful.

**Comment by naslars** [comment]
Hi, 

It seems like the issue is with the way the time difference is displayed on the profile. The time difference should be calculated based on the user's local time instead of UTC. To fix this issue, the design team should update the code to calculate the time difference based on the user's local time.

To make it more intuitive and useful, the design team can also consider displaying the time difference in a more readable format, such as "4 hours and 30 minutes ahead" instead of "5h30m ahead". This would make it easier for users to understand the time difference between their local time and the other user's time.

In addition, the design team can also consider adding an option for users to select their preferred time format, such as 12-hour or 24-hour format, and their preferred way of displaying the time difference. This would give users more control over how they view and interpret the time information on their profile.

**Reply by real-yfprojects** [reply]
I especially like your third point.
